modules/apps/object/object-rest-impl/rest-openapi.yaml

```diff
@@ -54,6 +54,13 @@ components:
                     type: string
                 scope:
                     $ref: "#/components/schemas/Scope"
+                thumbnailSrc:
```

Comment:
@Mikellorza, I'm not to fond of Src. Maybe use thumbnailLink or thumbnailHref? Src sound more of an internal name, and we are exposing this

Reply:
I am going to change the name from Src to URL and hope everything goes well.

Reply:
![](https://media0.giphy.com/media/v1.Y2lkPTc5MGI3NjExY2c5NDM5bmljcTR2dzEwZTV6a2RwYmluZG1qdzd3YjEycTB1NGhzbiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/St39CIobpWxPUmNUfH/giphy.gif)